CorsixTH/CorsixTH.lua

```diff
@@ -82,7 +82,7 @@ end
 
 -- Check Lua version
 if _VERSION ~= "Lua 5.1" then
-  if _VERSION == "Lua 5.2" or _VERSION == "Lua 5.3" then
+  if _VERSION == "Lua 5.2" or _VERSION == "Lua 5.3" or _VERSION == "Lua 5.4" then
```

Comment:
We may want to abort execution of the user has a wrong version of Lua 5.4, once we established that 5.4.2 works.